configs/config.go

```diff
@@ -0,0 +1,52 @@
+package configs
+
+import (
+	"io/ioutil"
+	"log"
+	"time"
+
+	yaml "gopkg.in/yaml.v3"
+)
+
+const ConfigurationPath = "/etc/gz-beat-shipper/gz-beat-shipper-conf.yml"
```

Comment:
My advise is to have this value be overridable somehow, with a CLI option or a configuration file, as this approach means that the user must have sudo rights in order to run the app.